src/tomography/rules.mk

```diff
@@ -65,6 +65,24 @@ tomography_MODULES = \
 	$(FC_MODDIR)/tomography_model_iso.$(FC_MODEXT) \
 	$(EMPTY_MACRO)
 
+postprocess_TARGETS = \
```

Comment:
For reference, I think the auxiliaries does this sort of split target definitions.